drop/template/Snakefile

```diff
@@ -10,24 +11,67 @@ drop.checkDropVersion(projectDir)
 _, projectPaths = drop.setupPaths(projectDir)
 tmp_dir = projectPaths["tmpDir"]
 
-cfg = drop.config.DropConfig(wbuild.utils.Config())
+cfg = drop.config.DropConfig(wbuild.utils.Config(), projectDir)
 drop.installRPackages(cfg)
 sa = cfg.sampleAnnotation
 config = cfg.config_dict # legacy
 
+def find_snakefile(module):
+    if os.path.isdir(module.getWorkdir()):
+        return module.getWorkdir() + "/Snakefile"
+    else:
+        return module.getWorkdir(hide_dir = True) + "/Snakefile"
 
+
+def moduleInputOutput(index_name, work_dir,module):
```

Comment:
rename to getModuleIndexFiles

Reply:
Move to Config class

Reply:
moved to submodule class. Because this happens at the sub module level